Build failure: pkgsCross.aarch64-multiplatform.lirc

### Steps To Reproduce

Steps to reproduce the behavior:
1. build *pkgsCross.aarch64-multiplatform.lirc*

### Build log

```
$ nix build nixpkgs#pkgsCross.aarch64-multiplatform.lirc -L
lirc-aarch64-unknown-linux-gnu> Running phase: unpackPhase
lirc-aarch64-unknown-linux-gnu> unpacking source archive /nix/store/67jfd8wfavi6x07nbf0lss6rcx4rrxbw-lirc-0.10.2.tar.bz2
lirc-aarch64-unknown-linux-gnu> source root is lirc-0.10.2
lirc-aarch64-unknown-linux-gnu> setting SOURCE_DATE_EPOCH to timestamp 1664870713 of file lirc-0.10.2/VERSION
lirc-aarch64-unknown-linux-gnu> Running phase: patchPhase
lirc-aarch64-unknown-linux-gnu> applying patch /nix/store/c0rlslgshl8q743fbwdijyrzqyc8i6c9-0001-Fix-Python-bindings.patch
lirc-aarch64-unknown-linux-gnu> patching file Makefile.am
lirc-aarch64-unknown-linux-gnu> Hunk #1 succeeded at 85 (offset 1 line).
lirc-aarch64-unknown-linux-gnu> patching file python-pkg/lirc/paths.py
lirc-aarch64-unknown-linux-gnu> applying patch /nix/store/g57jyrw8pd4lpjsyf0ip9x7vncp52mwm-linux-headers-5.18.patch
lirc-aarch64-unknown-linux-gnu> patching file daemons/lircd.cpp
lirc-aarch64-unknown-linux-gnu> patching file tools/lirc-lsplugins.cpp
lirc-aarch64-unknown-linux-gnu> patching script interpreter paths in .
lirc-aarch64-unknown-linux-gnu> ./missing: interpreter directive changed from "#! /bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./depcomp: interpreter directive changed from "#! /bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./config.sub: interpreter directive changed from "#!/usr/bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./python-pkg/lirc/lirctool: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./compile: interpreter directive changed from "#! /bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./tools/lirc-config-tool: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./tools/irtext2udp: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./tools/check_configs.py: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./tools/lircd-setup: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./tools/lirc-init-db: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./tools/pronto2lirc: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./tools/lirc-setup/lirc-setup: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./tools/lirc-make-devinput: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./tools/lirc-postinstall: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./tools/irdb-get: interpreter directive changed from "#!/usr/bin/env python3" to "/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3"
lirc-aarch64-unknown-linux-gnu> ./autogen.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./contrib/irman2lirc: interpreter directive changed from "#! /bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./install-sh: interpreter directive changed from "#!/bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./plugins/make-pluginlist.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./doc/make-driver-toc.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./doc/make-ext-driver-toc.sh: interpreter directive changed from "#!/usr/bin/env bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./doc/py_filter: interpreter directive changed from "#!/bin/bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./doc/fix-urls.sh: interpreter directive changed from "#!/usr/bin/bash" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/bash"
lirc-aarch64-unknown-linux-gnu> ./config.guess: interpreter directive changed from "#!/usr/bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> ./py-compile: interpreter directive changed from "#!/bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> substituteStream(): WARNING: pattern 'VERSION='0.0.0'' doesn't match anything in file 'python-pkg/setup.py'
lirc-aarch64-unknown-linux-gnu> Running phase: autoreconfPhase
lirc-aarch64-unknown-linux-gnu> autoreconf: export WARNINGS=
lirc-aarch64-unknown-linux-gnu> autoreconf: Entering directory '.'
lirc-aarch64-unknown-linux-gnu> autoreconf: configure.ac: not using Gettext
lirc-aarch64-unknown-linux-gnu> autoreconf: running: aclocal --force -I m4
lirc-aarch64-unknown-linux-gnu> autoreconf: configure.ac: tracing
lirc-aarch64-unknown-linux-gnu> autoreconf: running: libtoolize --copy --force
lirc-aarch64-unknown-linux-gnu> libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, '.'.
lirc-aarch64-unknown-linux-gnu> libtoolize: copying file './ltmain.sh'
lirc-aarch64-unknown-linux-gnu> libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
lirc-aarch64-unknown-linux-gnu> libtoolize: copying file 'm4/libtool.m4'
lirc-aarch64-unknown-linux-gnu> libtoolize: copying file 'm4/ltoptions.m4'
lirc-aarch64-unknown-linux-gnu> libtoolize: copying file 'm4/ltsugar.m4'
lirc-aarch64-unknown-linux-gnu> libtoolize: copying file 'm4/ltversion.m4'
lirc-aarch64-unknown-linux-gnu> libtoolize: copying file 'm4/lt~obsolete.m4'
lirc-aarch64-unknown-linux-gnu> autoreconf: configure.ac: not using Intltool
lirc-aarch64-unknown-linux-gnu> autoreconf: configure.ac: not using Gtkdoc
lirc-aarch64-unknown-linux-gnu> autoreconf: running: aclocal --force -I m4
lirc-aarch64-unknown-linux-gnu> autoreconf: running: /nix/store/iw4wi62kv0skc3dsgdmxbkpn45p7xk0b-autoconf-2.71/bin/autoconf --force
lirc-aarch64-unknown-linux-gnu> configure.ac:63: warning: The macro `AC_HEADER_STDC' is obsolete.
lirc-aarch64-unknown-linux-gnu> configure.ac:63: You should run autoupdate.
lirc-aarch64-unknown-linux-gnu> ./lib/autoconf/headers.m4:704: AC_HEADER_STDC is expanded from...
lirc-aarch64-unknown-linux-gnu> configure.ac:63: the top level
lirc-aarch64-unknown-linux-gnu> configure.ac:64: warning: The macro `AC_HEADER_TIME' is obsolete.
lirc-aarch64-unknown-linux-gnu> configure.ac:64: You should run autoupdate.
lirc-aarch64-unknown-linux-gnu> ./lib/autoconf/headers.m4:743: AC_HEADER_TIME is expanded from...
lirc-aarch64-unknown-linux-gnu> configure.ac:64: the top level
lirc-aarch64-unknown-linux-gnu> autoreconf: running: /nix/store/iw4wi62kv0skc3dsgdmxbkpn45p7xk0b-autoconf-2.71/bin/autoheader --force
lirc-aarch64-unknown-linux-gnu> autoreconf: running: automake --add-missing --copy --force-missing
lirc-aarch64-unknown-linux-gnu> configure.ac:20: installing './compile'
lirc-aarch64-unknown-linux-gnu> configure.ac:12: installing './missing'
lirc-aarch64-unknown-linux-gnu> Makefile.am: installing './depcomp'
lirc-aarch64-unknown-linux-gnu> autoreconf: Leaving directory '.'
lirc-aarch64-unknown-linux-gnu> Running phase: updateAutotoolsGnuConfigScriptsPhase
lirc-aarch64-unknown-linux-gnu> Updating Autotools / GNU config script to a newer upstream version: ./config.sub
lirc-aarch64-unknown-linux-gnu> Updating Autotools / GNU config script to a newer upstream version: ./config.guess
lirc-aarch64-unknown-linux-gnu> Running phase: updateAutotoolsGnuConfigScriptsPhase
lirc-aarch64-unknown-linux-gnu> Updating Autotools / GNU config script to a newer upstream version: ./config.sub
lirc-aarch64-unknown-linux-gnu> Updating Autotools / GNU config script to a newer upstream version: ./config.guess
lirc-aarch64-unknown-linux-gnu> Running phase: configurePhase
lirc-aarch64-unknown-linux-gnu> fixing libtool script ./ltmain.sh
lirc-aarch64-unknown-linux-gnu> ./configure
lirc-aarch64-unknown-linux-gnu> patching script interpreter paths in ./configure
lirc-aarch64-unknown-linux-gnu> ./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh"
lirc-aarch64-unknown-linux-gnu> configure flags: --disable-static --disable-dependency-tracking --prefix=/nix/store/p0l924n7sxgxkna5gqp05mvyz8cs8w80-lirc-aarch64-unknown-linux-gnu-0.10.2 --sysconfdir=/etc --localstatedir=/var --with-systemdsystemunitdir=\$\(out\)/lib/systemd/system --enable-uinput --enable-devinput --with-lockdir=/run/lirc/lock PYTHON=/nix/store/7lzksh1xlh5ynfscsnjq5wdydi6wiq9f-python3-3.11.6-env/bin/python3.11 --build=x86_64-unknown-linux-gnu --host=aarch64-unknown-linux-gnu
lirc-aarch64-unknown-linux-gnu> checking for a BSD-compatible install... /nix/store/zx8aqgdy735qzk81glfwil6mbi6ddqb1-coreutils-9.4/bin/install -c
lirc-aarch64-unknown-linux-gnu> checking whether build environment is sane... yes
lirc-aarch64-unknown-linux-gnu> checking for aarch64-unknown-linux-gnu-strip... aarch64-unknown-linux-gnu-strip
lirc-aarch64-unknown-linux-gnu> checking for a race-free mkdir -p... /nix/store/zx8aqgdy735qzk81glfwil6mbi6ddqb1-coreutils-9.4/bin/mkdir -p
lirc-aarch64-unknown-linux-gnu> checking for gawk... gawk
lirc-aarch64-unknown-linux-gnu> checking whether make sets $(MAKE)... yes
lirc-aarch64-unknown-linux-gnu> checking whether make supports nested variables... yes
lirc-aarch64-unknown-linux-gnu> checking for aarch64-unknown-linux-gnu-gcc... aarch64-unknown-linux-gnu-gcc
lirc-aarch64-unknown-linux-gnu> checking whether the C compiler works... yes
lirc-aarch64-unknown-linux-gnu> checking for C compiler default output file name... a.out
lirc-aarch64-unknown-linux-gnu> checking for suffix of executables...
lirc-aarch64-unknown-linux-gnu> checking whether we are cross compiling... yes
lirc-aarch64-unknown-linux-gnu> checking for suffix of object files... o
lirc-aarch64-unknown-linux-gnu> checking whether the compiler supports GNU C... yes
lirc-aarch64-unknown-linux-gnu> checking whether aarch64-unknown-linux-gnu-gcc accepts -g... yes
lirc-aarch64-unknown-linux-gnu> checking for aarch64-unknown-linux-gnu-gcc option to enable C11 features... none needed
lirc-aarch64-unknown-linux-gnu> checking whether aarch64-unknown-linux-gnu-gcc understands -c and -o together... yes
lirc-aarch64-unknown-linux-gnu> checking whether make supports the include directive... yes (GNU style)
lirc-aarch64-unknown-linux-gnu> checking dependency style of aarch64-unknown-linux-gnu-gcc... none
lirc-aarch64-unknown-linux-gnu> checking whether the compiler supports GNU C++... yes
lirc-aarch64-unknown-linux-gnu> checking whether aarch64-unknown-linux-gnu-g++ accepts -g... yes
lirc-aarch64-unknown-linux-gnu> checking for aarch64-unknown-linux-gnu-g++ option to enable C++11 features... none needed
lirc-aarch64-unknown-linux-gnu> checking dependency style of aarch64-unknown-linux-gnu-g++... none
lirc-aarch64-unknown-linux-gnu> checking for sh... /nix/store/q8qq40xg2grfh9ry1d9x4g7lq4ra7n81-bash-5.2-p21/bin/sh
lirc-aarch64-unknown-linux-gnu> checking for modinfo... no
lirc-aarch64-unknown-linux-gnu> configure: WARNING: "No modinfo command found - skipping kernel drivers."
lirc-aarch64-unknown-linux-gnu> checking for pkg-config... no
lirc-aarch64-unknown-linux-gnu> configure: error: Required program pkg-config is missing or not in $PATH
```

### Additional context

Cross-compiling as a dependency of Kodi.

Possibly related to https://github.com/NixOS/nixpkgs/pull/190212.

### Notify maintainers

@pSub 

### Metadata

~Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.~
Nixpkgs revision: `d6863cbcbbb80e71cecfc03356db1cda38919523`

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/libraries/lirc/default.nix
+++ b/pkgs/development/libraries/lirc/default.nix
@@ -62,6 +62,8 @@ stdenv.mkDerivation rec {
   preConfigure = ''
     # use empty inc file instead of a from linux kernel generated one
     touch lib/lirc/input_map.inc
+
+    export PKGCONFIG="$PKG_CONFIG"
   '';
 
   strictDeps = true;

